seqr/views/utils/test_utils.py

```diff
@@ -655,54 +663,62 @@ def assert_airflow_calls(self, trigger_error=False, additional_tasks_check=False
         dag_variable_overrides = self._get_dag_variable_overrides(additional_tasks_check)
         dag_variables = {
             'projects_to_run': [dag_variable_overrides['project']] if 'project' in dag_variable_overrides else self.PROJECTS,
-            'callset_path': f'gs://test_bucket/{dag_variable_overrides["callset_path"]}',
-            'sample_type': dag_variable_overrides['sample_type'],
             'dataset_type': dataset_type or dag_variable_overrides['dataset_type'],
             'reference_genome': dag_variable_overrides.get('reference_genome', 'GRCh38'),
+            'callset_path': f'gs://test_bucket/{dag_variable_overrides["callset_path"]}',
+            'sample_type': dag_variable_overrides['sample_type'],
         }
         if dag_variable_overrides.get('skip_validation'):
             dag_variables['skip_validation'] = True
         dag_variables['sample_source'] = dag_variable_overrides['sample_source']
-        self._assert_airflow_calls(dag_variables, call_count, offset=offset)
+        self._dag_variables = dag_variables
+        self._assert_airflow_calls(call_count, offset=offset)
```

Comment:
rather than having `self._dag_variables` being constantly  overwritten it would be better for  `_assert_airflow_calls` to take `dag_variables` as an argument